ui/src/timeMachine/actions/index.ts

```diff
@@ -100,6 +100,11 @@ export type Action =
   | ReturnType<typeof updateCheckThreshold>
   | ReturnType<typeof removeCheckThreshold>
 
+type ExternalActions =
+  | ReturnType<typeof loadBuckets>
+  | ReturnType<typeof saveAndExecuteQueries>
+  | ReturnType<typeof setValues>
```

Comment:
do all the the functions that rely on external actions call all these functions, or is this just the set of all the external functions?

Reply:
"`ExternalActions`" was just a name I thought of for actions not declared in this file.  Really, I want a central repository and format for all action types.